modelbaker/utils/db_utils.py

```diff
@@ -152,3 +153,41 @@ def db_ili_version(configuration):
     db_connector = get_db_connector(configuration)
     if db_connector:
         return db_connector.ili_version()
+
+
+def get_service_names():
+    """
+    Provides the available service_names if
+    """
+    try:
+        return pgserviceparser.service_names(), None
+    except pgserviceparser.ServiceFileNotFound as sfe:
+        return (
+            [],
+            f"The last used service {servicename} cannot be found, since no service file {str(sfe)} available anymore.",
+        )
+    except pgserviceparser.ServiceNotFound:
+        return (
+            [],
+            f"The last used service {servicename} cannot be found in the service file.",
```

Comment:
I think these strings will fail to be formatted because `servicename` does not exist in this context.